File: CodeForces/TLE/800/ArrayColoring.java

import java.util.Scanner;

public class ArrayColoring {
    static Scanner sc = new Scanner(System.in);

    static void solve(){
        int n = sc.nextInt();
        int arr[]= new int[n];
        int sum=0;

        for(int i=0;i<n;i++){
            arr[i] = sc.nextInt();
        }
        for (int i = 0; i < arr.length; i++) {
            sum+=arr[i];
        }

        if(sum%2==0) System.out.println("YES");
        else System.out.println("NO");
    }
    public static void main(String[] args) {
        int t = sc.nextInt();
        while (t-->0) {
           solve(); 
        }        
    }
    
}

File: CodeForces/TLE/800/ArrayColoring.cpp
#include <iostream>
#include <vector>

using namespace std;

void solve() {
    int n;
    cin >> n;
    vector<int> arr(n);
    int sum = 0;
    for (int i = 0; i < n; i++) {
        cin >> arr[i];
    }
    for (int i = 0; i < arr.size(); i++) {
        sum += arr[i];
    }
    if (sum % 2 == 0) {
        cout << "YES" << endl;
    } else {
        cout << "NO" << endl;
    }
}

int main(){
    int t;
    cin>>t;
    while(t-->0){
        solve();
    }
}